crates/cargo-test-support/src/compare.rs

```diff
@@ -16,6 +16,11 @@
 //!   `[WARNING]`) to match cargo's "status" output and allows you to ignore
 //!   the alignment. See the source of `substitute_macros` for a complete list
 //!   of substitutions.
+//! - `[DIRTY-MSVC]` (only when the line starts with it) would be replaced by
```

Comment:
There were so many tests that actually used it (see ef581f3fbf2f1a8c74f943fc186af9e3a0160897), but it's not used anymore since `Dirty` is only displayed with `--verbose`. Some new tests with `--verbose` will need it for sure.

Reply:
Is there any other test MSVC specific? If there is only a few, I recommend not using, as replacing the output string might be a bit excessive. (I didn't benchmark though, but test suite is already slow).

Reply:
Pretty much all the tests that were modified in ef581f3fbf2f1a8c74f943fc186af9e3a0160897, counting up to around 13

Reply:
Fwiw `str::replace` is called 44 times during `substitute_macros`:

https://github.com/rust-lang/cargo/blob/7fb01c68c190c28a43581ae7c719dcf057ecc002/crates/cargo-test-support/src/compare.rs#L158-L209

Reply:
Make sense. Thank you!